src/components/FiltersPanel/FiltersPanel.js

```diff
@@ -0,0 +1,134 @@
+import React from 'react'
+import { useFilter } from '../../contexts/filterContext'
+
+ function FiltersPanel() {
+    const { filterState ,filterDispatch} = useFilter()
+  return (
+    <>
+        <main>
```

Comment:
no need for fragment, since you have a main tag and there's nothing outside that